draft-ietf-moq-transport.md

```diff
@@ -1074,33 +1075,22 @@ GOAWAY Message {
 ## SUBSCRIBE {#message-subscribe-req}
 
 ### Filter Types {#sub-filter}
+The subscriber specifies a filter on the subscription to allow the publisher to identify which objects need to be delivered.
 
-The subscriber specifies a filter on the subscription to allow
-the publisher to identify which objects need to be delivered.
-
-There are 4 types of filters:
+There are 3 types of filters:
 
-Latest Group (0x1) : Specifies an open-ended subscription with objects
-from the beginning of the current group.  If no content has been delivered yet,
-the subscription starts with the first published or received group.
+Latest Group (0x1) : Specifies an open-ended subscription with objects from the beginning of the current group.
```

Comment:
This is no longer subscribe to live edge, it is fetch at line rate to reach live edge, then subscribe at media rate to live edge. I think this clouds the proposed separation of fetch vs subscribe. It will be hard to defend the separation with this.

Reply:
My understanding of the live edge is group boundary and MOQ helps solve lot of unnecessary key frame requests due to its caching my joiners are vwey close to join time of each other.  
If some one wants more live than that, they can go to the current object or can wait for the next group